docs/ref/contrib/sitemaps.txt

```diff
@@ -186,6 +186,15 @@ Note:
         Django respond appropriately to requests with an ``If-Modified-Since``
         header which will prevent sending the sitemap if it hasn't changed.
 
+    .. property:: Sitemap.paginator
+
+        **Optional.**
+
+        If you generate multiple sitemaps in a batch, then it makes sense to
+        overwrite this property. You could for example use a ``cached_property``.
+        Otherwise the ``items()`` method gets called several times, which
+        wastes computing resources.
```

Comment:
Maybe :thinking: :
```
        This property return a :class:`~django.core.paginator.Paginator` for
        :attr:`~Sitemap.items()`. If you generate sitemaps in a batch you may
        want to override this as a cached property in order to avoid multiple
        ``items()`` calls.
```